### DISCUSSION TITLE
problemas con la licencia

### DISCUSSION THREAD

**Discussion by brujillaverde** [aclap-dev/video-downloadhelper#461] [Help and Q&A]
### Your question

Buenas tardes
Hace varias semanas compré la licencia. Me funcionaba perfectamente pero, de repente, no podía descargar videos de classroom. Actualicé la coapp y, desde ese momento, me salta de manera repetida la pantalla que solicita el permiso para que la coapp haga cambios en mi pc.
Además, no puedo descargar videos.
¿Podríais ayudarme?
Tengo windows 11 y descargué la coapp de 64

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: es-ES
license: nocoapp
key (only 16 first characters): 2f6786d6eebd4bf6
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
coapp: no.

**Comment by jcfrog** [comment]
Hi,
Did you accept the CoApp modifications?
I guess these videos are not public

**Comment by brujillaverde** [comment]
Si, claro que lo acepté.
Pero inmediatamente, volvía a pedirme lo mismo, que aceptara los cambios.

**Comment by brujillaverde** [comment]
Efectivamente los vídeos no son públicos

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

Your information above says you do not have the CoApp installed.

> Coapp: No.

Since your content is private, nobody here can determine whether installing the CoApp would be helpful.  Instructions for installing are located in the Wiki.  Click the Wiki link at the top of this page, then read that page carefully.  You'll find a link to instructions for installing the CoApp.

In the case of private content, outsiders can sometimes -- SOMETIMES -- get the private content anyway.  It involves posting Hit Details here.  In this discussion #391 the user there was trying to post his Hit Details.  There are instructions there for how to find the Hit Details.  Be sure you click the link that says, "Show 5 previous replies."  A few posts after those instructions, you will find more instructions on what to do after you find the Hit Details.  A simple Copy/Paste does not work.  Be sure you read #391 carefully.  The things you need to do are not all together in one place.  Once we have your Hit Details, it is possible we might be able to help you.  POSSIBLE.  Not guaranteed.  Just possible.

**Reply by GrampaWildWilly** [reply]
In addition, simply installing the CoApp on your system is not enough to get VDH to actually use it.  This is a topic that is discussed here: #440.
